README.md

```diff
@@ -15,19 +15,29 @@ Then install the application by going to cftool directory and typing:
 
 `./build.sh`
 
-Configuration file (config.yaml) will be copied to `/etc/.Appliscale/cftool`
+Configuration file (config.yaml) will be copied to your home directory under the `/.Appliscale/cftool/config.yaml` path.
```

Comment:
This should be dependent on the _OS_ - e.g. on _Linux_ you should use `/etc`, on Windows different and so on.
Also I don't see a point of adding `.Appliscale` for it - let's use `~/.config/cftool`.

And here I think there is a lack of `~` indicating that it is a home directory.

But let's leave it as a another exercise - either let's create a task for that or a note to the existing task and do not close it yet.